packages/react/kibana_context/root/root_provider.tsx

```diff
@@ -15,6 +16,8 @@ import { KibanaEuiProvider, type KibanaEuiProviderProps } from './eui_provider';
 export interface KibanaRootContextProviderProps extends KibanaEuiProviderProps {
   /** The `I18nStart` API from `CoreStart`. */
   i18n: I18nStart;
+  /** The `AnalyticsServiceStart` API from `CoreStart`. */
+  analytics?: AnalyticsServiceStart;
```

Comment:
Optional, but in reality is not provided from anywhere, to avoid making breaking changes for maintainers and to minimize the changes in this PR.